circuit_knitting/cutting/cut_wire_to_move.py

```diff
@@ -0,0 +1,82 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2023.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Function to transform a :class:`.CutWire` instruction to a :class:`.Move` instruction."""
+from __future__ import annotations
+
+from itertools import groupby
+from qiskit.circuit import Qubit, QuantumCircuit
+from circuit_knitting.cutting.instructions.move import Move
+
+
+def transform_to_move(circuit: QuantumCircuit) -> QuantumCircuit:
+    """Transform a :class:`.CutWire` instruction to a :class:`.Move` instruction.
+
+    Args:
+        circuit (QuantumCircuit): original circuit with :class:`.cut_wire` instructions.
+
+    Returns:
+        circuit (QuantumCircuit): new circuit with :class`.move` instructions.
+    """
+    new_circuit, mapping = _circuit_structure_mapping(circuit)
+
+    for instructions in circuit.data:
+        gate_index = [circuit.find_bit(qubit).index for qubit in instructions.qubits]
+
+        if instructions in circuit.get_instructions("cut_wire"):
```

Comment:
`circuit.get_instructions("cut_wire")` can probably be called only once, outside the loop, I think.

Reply:
or, you could even just look at the current instruction to see if it is a `Move` or not